Sources/UI/Web View/WebViewController.swift

```diff
@@ -37,6 +37,18 @@ open class WebViewController: UIViewController, WKNavigationDelegate {
         }
     }
     
+    override open func present(_ viewControllerToPresent: UIViewController, animated flag: Bool, completion: (() -> Void)? = nil) {
+        setUIDocumentMenuViewControllerSoureViewsIfNeeded(viewControllerToPresent)
+        super.present(viewControllerToPresent, animated: flag, completion: completion)
+    }
+    
+    func setUIDocumentMenuViewControllerSoureViewsIfNeeded(_ viewControllerToPresent: UIViewController) {
```

Comment:
Apple slowly moving from removing `UI` prefix in UIKit, `UIBundle` -> `Bundle` and etc. I think we can omit this prefix in our methods. It's pretty clear what is happening. It's up to you to change it.